### DISCUSSION TITLE
مزیت هدیفای دسکتاپ

### DISCUSSION THREAD

**Discussion by sh4msi** [hiddify/Hiddify-Manager#214] [Q&A سوالات رایج]
سلام
ممنون از زحماتی که می کشید
مزیت Hiddify Desktop نسبت به Clash Verge چی هست؟ 
یا قراره در اینده چه مزیتی داشته باشه؟

چون بصورت کلی اگر مزیتی نداشته باشه عموم استفاده از سورس اصلی خیلی بهتره

**Comment by hiddify-com** [comment]
اولین چیزی که هست بای دیفالت از کور کلش متا استفاده میکنه
دوم اینکه کلیک میکنی روی لینک، مستقیم توی برنامه باز میشه
سوم وقتی لینک ادد میکنی مستقیم انتخاب میکنه
چهارم، اسم درست حسابی میذاره برای کانفیگ
پنجم ریدیزان شده قسمت های مهم در دسترس تر باشن
به زودی چیزای دیگه هم اضافه میشه:دی

**Reply by sh4msi** [reply]
یکی از کارهای خوب هم میتونه تغییر رنگ ایکن برنامه در زمان فعال بودن باشه

**Comment by raminrez** [comment]
Clash Verge و Hiddify Desktop هیچکدوم روی مک جواب نمیدن
دلیلشم اینه که ClashMeta-Core ارور میده و انتخاب نمیشه

**Comment by esmaeilifardi** [comment]
> سلام ممنون از زحماتی که می کشید مزیت Hiddify Desktop نسبت به Clash Verge چی هست؟ یا قراره در اینده چه مزیتی داشته باشه؟
> 
> چون بصورت کلی اگر مزیتی نداشته باشه عموم استفاده از سورس اصلی خیلی بهتره

دو بار روی اوبونتو نسخه آخر خام نصب کردم بار اول خطای ssl برای nginx داد بار دون خطای ثبت دامنه داد در نهایت مجددا برگشت روی نسخه اصلی و پاکش کردم گیرو گور زیاد داره
